docs/source/azure/configuration/helmtutorial.rst

```diff
@@ -4,4 +4,194 @@ Tutorial - Helm Deployment
+---------------------------
+
+This storage account will hold all of the blobs and queues used by the container host. You could 
+also use an existing resource here or multiple storage accounts.
+
+We will also create a blob container. One will host the uploaded policies, and the other will store
+the custodian output files of running policies.
+
+.. code-block:: bash
+
+    az storage account create --resource-group c7n-helm-tutorial --name c7nstorage
+
+    account_key=$(az storage account keys list --account-name c7nstorage --query "[0].value" --output tsv)
+    az storage container create --account-name c7nstorage --account-key $account_key --name c7n-helm-policies
+    az storage container create --account-name c7nstorage --account-key $account_key --name c7n-helm-logs
+
+3. Create a Service Principal
+-----------------------------
+
+This service principal will be used as the identity for the container host. You could also use an existing service principal.
+Remember to save the password to use as the client secret.
+
+.. code-block:: bash
+
+    az ad sp create-for-rbac --name c7n-helm 
+
+This service principal will need the proper permissions to interact with the storage account and any other 
+azure resources that the Azure Container Host interacts with. This includes the resources that 
+policies will run against.
+
+The simplest way to grant these permissions is to make the identity a ``Contributor`` on the target 
+subscription and a ``Storage Blob Data Contributor`` and ``Storage Queue Data Contributor`` on the 
+storage account created above.
+
+.. code-block:: bash
+
+    az role assignment create --assignee <Service Principal Client Id> \
+        --role "Contributor" --scope <Target Subscription Resource Id>
+
+    az role assignment create --assignee <Service Principal Client Id> \
+        --role "Storage Blob Data Contributor" --scope <Storage Account Resource Id>
+
+    az role assignment create --assignee <Service Principal Client Id> \
+        --role "Storage Queue Data Contributor" --scope <Storage Account Resource Id>
+
+4. Create an Application Insights Instance
+------------------------------------------
+
+We will point the container host to this Application Insights to gather logs and monitor the host
+and running policies.
+
+.. code-block:: bash
+
+    # You may need to add the application-insights extension
+    az extension add -n application-insights
+
+    az monitor app-insights component create --resource-group c7n-helm-tutorial --app c7n-helm-insights --location westus2    
+
+5. Create an AKS Cluster and Install Tiller
+-------------------------------------------
+
+This AKS cluster will be used to host the pods running the Container Host. Tiller will also need to 
+be installed with the proper service account.
+
+.. code-block:: bash
+
+    az aks create --resource-group c7n-helm-tutorial --name c7n-helm \
+        --service-principal <Service Principal ID> --client-secret <Client Secret>
```

Comment:
I would explicitly specify the node-count and the node-vm-size so people don't accidentally provision a few hundred dollars a month cluster